resources/views/account/partials/address-cards-popup.blade.php

```diff
@@ -30,6 +30,12 @@ class="w-full sm:min-w-[350px]"
                     <x-rapidez-ct::button.link v-else v-on:click="userAddress.default_billing=false;mutate()">
                         @lang('Deselect as billing')
                     </x-rapidez-ct::button.link>
+                    <graphql-mutation query="mutation deleteCustomerAddress($id: Int!){ deleteCustomerAddress ( id: $id ) }" :variables="{ id: userAddress.id }" v-slot="{ mutate }">
```

Comment:
Now we always have text for the delete button. Can you make it more flexible so that we are able to change text to a delete icon if we would like that?